chandra_aca/tests/test_maude_decom.py

```diff
@@ -1185,3 +1185,29 @@ def test_filter_vcdu_jumps():
     for case in filter_vcdu_test_cases:
         idx = maude_decom.filter_vcdu_jumps(case["inp"])
         assert np.all(idx == case["exp"])
+
+
+def test_end_integ_time():
+    from astropy import table
+
+    start, stop = (686111020, 686111030)
+    table_l0 = maude_decom.get_aca_packets(start, stop, adjust_time=True)
+    table_maude = maude_decom.get_aca_packets(start, stop, adjust_time=False)
+    comparison = table.join(
+        table_l0, table_maude, table_names=["l0", "maude"], keys=["VCDUCTR", "IMGNUM"]
+    )
```

Comment:
Can you help me understand the need for this inner join? It seems it could potentially hide a problem because it could be throwing out rows that don't match. My expectation is that `adjust_time=True or False` will *only* change `table["TIME"]`, so you can simply check:
```
np.allclose(table_l0["END_INTEG_TIME"], table_maude["END_INTEG_TIME"], rtol=0)
```
This matches the pattern of next two tests (but note that `np.allclose()` is your friend here).

Reply:
> Can you help me understand the need for this inner join?

When you run with `adjust_time=True`, it adds padding around the time, and the final result is filtered according to the adjusted times. If you call the function with the same start/stop times, you will end up with a different set of images.

The alternative is to call each function with different values of start/stop to compensate for the time adjustment. I can do that.

Reply:
I changed the test so the arguments for each function call are shifted as expected, so a join is not necessary anymore.